### PR TITLE
Generalize capsule and common playbooks

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This aims to do two things:

 1) organize according to Ansible best practices [1]
 2) Generalize common and capsule roles away from the specialized Vagrant grouping

[1] http://docs.ansible.com/ansible/playbooks_best_practices.html#directory-layout

File: ansible.cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [defaults]
-callback_plugins = playbooks/callback_plugins/
+callback_plugins = ./ansible/callback_plugins/
```
Comment (stbenjam) [inline ansible.cfg]: Is this correct? Looks like callback_plugins are in playbooks directory.

Comment (ehelms) [inline ansible.cfg]: Found my leftovers.

**Comment by ehelms** [comment]
Updated

**Comment by stbenjam** [comment]
Thanks, ACK from me

**Pull request merged**